package.json

```diff
@@ -86,7 +86,7 @@
     "@mnajdova/enzyme-adapter-react-18": "^0.2.0",
     "@mui/icons-material": "^5.15.7",
     "@mui/material": "^5.15.7",
-    "@mui/monorepo": "https://github.com/mui/material-ui#v5.15.7",
+    "@mui/monorepo": "github:mui/material-ui#v5.15.7",
```

Comment:
Installing locally without a cache was failing with not a git repo error. 🙈

Reply:
I have no idea what are the implication of this change tbh :grimacing:

Reply:
It's the same as in `master`: 
https://github.com/mui/mui-x/blob/5f6ceb4a0f54d915c75dd8df097491360a7ea418/package.json#L92

It allows to reliably target tag/branch dependencies as compared to https url, which is default branch reliant. 🙈